src/magic_folder/snapshot.py

```diff
@@ -420,15 +426,17 @@ def create_snapshot_from_capability(snapshot_cap, tahoe_client):
         snapshot_json = yield tahoe_client.download_capability(snapshot_cap)
         snapshot = json.loads(snapshot_json)[1]["children"]
 
-        # create SnapshotAuthor
-        author_cap = snapshot["author"][1]["ro_uri"]
-        author_json = yield tahoe_client.download_capability(author_cap)
-        snapshot_author = json.loads(author_json)
-
-        author = create_author_from_json(snapshot_author)
+        # NB: once we communicate authors + public-keys some other
+        # way, we could iterate all our known authors and attempt to
+        # verify a signature against the snapshot + metdata
+        # capabilities at this point .. that might be better anyway?
+        # (A key advantage is not even trying to deserialize anything
+        # that's not verified by a signature).
```

Comment:
Perhaps we could put the author pubkey into the snapshot as well so we don't have to try every known author, we can just look up the one with the matching key.

Or maybe looking up the author by the "name" field is good enough.

Reply:
The author _is_ in the snapshot currently (including pubkey). However, there is no way to **verify** this key yet. The "real" solution is to communicate the keys along with participants..